AdobeIms/Model/UserProfileRepository.php

```diff
@@ -91,7 +91,7 @@ public function get(int $entityId): UserProfileInterface
         $entity = $this->entityFactory->create();
         $this->resource->load($entity, $entityId, self::ID);
         if (!$entity->getId()) {
-            $message = __('User profile with id %id not found.', ['id' => $entityId]);
+            $message = __('User profile with id %id not found.', [self::ID => $entityId]);
```

Comment:
This is a mistake, array key is applied to '%id' in there string nearby, while ID constant is a column to load data.

Reply:
@sivaschenko by the way, I think `ID` constant should not be used in ->load call above. Data interface constant should be used instead.